catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/AbstractCswSource.java

```diff
@@ -947,6 +959,23 @@ public ResourceResponse retrieveResource(URI resourceUri,
         return resourceReader.retrieveResource(resourceUri, requestProperties);
     }
 
```

Comment:
It's unfortunately to need to duplicate this sort of utility code everywhere. Is there nowhere we could centralize it?